bin/load_scxa_analytics_index.sh

```diff
@@ -10,4 +10,19 @@ export ONTOLOGY_PROCESSOR=$SOLR_COLLECTION\_ontology_expansion
 
 echo "Loading cond. sdrf $CONDENSED_SDRF_TSV into host $SOLR_HOST collection $SOLR_COLLECTION..."
 
-condSdrf2tsvForSCXAJSONFactorsIndex.sh $CONDENSED_SDRF_TSV | jsonFilterEmptyFields.sh | loadJSONIndexToSolr.sh
+CHUNK_PREFIX=${CHUNK_PREFIX:-"cond-sdrf-chunk-"}
+NUM_DOCS_PER_BATCH=${NUM_DOCS_PER_BATCH:-100000}
+split -a 3 -l $NUM_DOCS_PER_BATCH $CONDENSED_SDRF_TSV $CHUNK_PREFIX
+CHUNK_FILES=`ls $CHUNK_PREFIX*`
+
+I=O
+for CHUNK_FILE in $CHUNK_FILES
+do
+  I=$(($I + 1)) 
+  echo "$CHUNK_FILE ${I}/`wc -w <<< $CHUNK_FILES`"
+  condSdrf2tsvForSCXAJSONFactorsIndex.sh $CHUNK_FILE | jsonFilterEmptyFields.sh | loadJSONIndexToSolr.sh
```

Comment:
we might need to change the set -e I guess before the loop, otherwise I think it will just die if before the break and all.